### PR TITLE
[IT-3456] Add Windows Server 2022 AMI

### PR DESCRIPTION
**Pull request by ConsoleCatzirl**
Initial template, largely based off of `Sage-Bionetworks-IT/packer-base-winserver2019`.

### CODE REVIEW

File: src/template.json [hunk 1]
```diff
@@ -1,49 +1,41 @@
...
         "most_recent": true,
         "owners": [
-          "099720109477"
+          "amazon"
         ]
       },
-      "ssh_username": "{{user `SshUsername`}}",
       "tags": {
         "BuiltBy": "Packer",
+        "Department": "{{user `Department`}}",
```
Comment (zaro0508) [inline src/template.json]: why are `Department` and `Project` tags added?  i thought we are only using `CostCenter` tag now?

File: src/boot_config/winrm_bootstrap.txt [hunk 2]
```diff
@@ -0,0 +1,47 @@
+<powershell>
```
Comment (zaro0508) [inline src/boot_config/winrm_bootstrap.txt:1]: did you come up with this file on your own or did you copy it from somewhere?  if copied it from somewhere could you please cite the source?

**Review by zaro0508 — CHANGES_REQUESTED** [review]

File: src/template.json (hunk 1 above)
Comment (ConsoleCatzirl) [reply src/template.json]: whoops, I should've removed those after copying the template from https://github.com/Sage-Bionetworks-IT/packer-base-winserver2019/blob/v0.0.1/src/template.json

File: src/boot_config/winrm_bootstrap.txt (hunk 2 above)
Comment (ConsoleCatzirl) [reply src/boot_config/winrm_bootstrap.txt:1]: I copied this from https://github.com/Sage-Bionetworks-IT/packer-base-winserver2019/blob/v0.0.1/src/boot_config/winrm_bootstrap.txt and added a comment with that link

Comment (zaro0508) [reply src/boot_config/winrm_bootstrap.txt:1]: ahh, actually you should probably cite from here https://developer.hashicorp.com/packer/integrations/hashicorp/amazon/latest/components/builder/ebs#connecting-to-windows-instances-using-winrm

**Review by zaro0508 — APPROVED** [review]
LGTM, but i would update the citation.

File: src/boot_config/winrm_bootstrap.txt (hunk 2 above)
Comment (ConsoleCatzirl) [reply src/boot_config/winrm_bootstrap.txt:1]: oh, awesome, thanks for the better link

**Pull request merged**